webapp/views.py

```diff
@@ -225,6 +225,46 @@ def advantage_view():
     )
 
 
+def post_stripe_method_id():
+    if auth.is_authenticated(flask.session):
+        if not flask.request.is_json:
+            return flask.jsonify({"error": "JSON required"}), 400
+
+        payment_method_id = flask.request.json.get("payment_method_id")
+        if not payment_method_id:
+            return flask.jsonify({"error": "payment_method_id required"}), 400
+
+        account_id = flask.request.json.get("account_id")
+        if not account_id:
+            return flask.jsonify({"error": "account_id required"}), 400
+
+        return advantage.put_method_id(
+            flask.session, account_id, payment_method_id
+        )
+    else:
+        return flask.jsonify({"error": "authentication required"}), 401
+
+
+def get_renewal(renewal_id=""):
```

Comment:
`renewal_id` is always going to be set right? otherwise the url rule would not have matched?
same for the view bellow.